README.md

```diff
@@ -14,6 +14,12 @@ Lock for Android
 * Provides support for **Social Providers** (Facebook, Twitter, etc.), **Enterprise Providers** (AD, LDAP, etc.) and **Username & Password**.
 * Passwordless authentication using **SMS** and **Email**.
 
+
+## Notes On Embedded Authentication
+
+Since June 2017 new Clients no longer have the **Password Grant Type*** enabled by default. This means that Database Authentication from *untrusted clients* like a mobile app, is not possible until the Grant Type is enabled again. The reasons of this change are listed in this [embedded vs native mobile article](https://auth0.com/docs/tutorials/browser-based-vs-native-experience-on-mobile). If you choose to use this library anyway, follow [this guide](https://auth0.com/docs/clients/client-grant-types#how-to-edit-the-client-grant_types-property) to learn how to enable the Password Grant Type.
```

Comment:
~like a mobile app, is not possible until the Grant Type is enabled again.~ -> such as a mobile app, require this Grant Type to be enabled.

~The reasons of this change~ -> The reasons for this change

~If you choose to use this library anyway, follow~ ->  Follow